### PR TITLE
Hide display parameter for mail context

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
Hides the display parameter for Contact -> Mail custom fields.

See more details in #15915.

### Testing Instructions
Create a new Contact -> Mail custom field.

### Expected result
The "Automatic Display" field in options is not displayed.

### Actual result
The "Automatic Display" field in options is displayed.

### CODE REVIEW

File: administrator/components/com_contact/models/forms/fields/mail.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<fields name="params" label="COM_FIELDS_FIELD_BASIC_LABEL">
+		<fieldset name="basic">
+			<field
+				name="display"
+				type="hidden"
+			    default="2"
```
Comment (Quy) [inline administrator/components/com_contact/models/forms/fields/mail.xml]: Indent

**Comment by laoneo** [comment]
@Quy thanks for the hint. Done.

**Comment by AlexRed** [comment]
I have tested this item :white_check_mark: successfully on 1a63890e6299969d9eb817d8d61a02559b827f95<br /><br />Patch ok for me. Thanks  :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15953">issues.joomla.org/tracker/joomla-cms/15953</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 1a63890e6299969d9eb817d8d61a02559b827f95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15953">issues.joomla.org/tracker/joomla-cms/15953</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**